pkg/apis/enterprise/v1/common_types.go

```diff
@@ -272,6 +271,18 @@ type AppSrcDeployInfo struct {
 	AppDeploymentInfoList []AppDeploymentInfo `json:"appDeploymentInfo,omitempty"`
 }
 
+// AppInfoStatus indicates if bundle push required
+type AppInfoStatus struct {
+	// This is false for the first time when we enter the reconcile loop.
+	// For all other consequent times, this will be set to true.
+	// This will help in checking for the apps status either right away(for the first time case)
+	// or after the polling interval.
```

Comment:
Does this mean its used for determining whether the app list get is cause by the initial kubectl apply of the CR or the app interval timer popping?

Reply:
Yeah you are right. But as I think about it and also as Subba has mentioned, we probably won't need this variable now.